app/handler/commands.js

```diff
@@ -552,6 +553,71 @@ exports.handleMessage = function (steamID, message) {
         });
     } else if (isAdmin && command === 'update') {
         const params = getParams(message.substring(command.length + 1).trim());
+        
+        if (typeof params.intent === 'string') {
+            const intent = ['buy', 'sell', 'bank'].indexOf(params.intent.toLowerCase());
+            if (intent !== -1) {
+                params.intent = intent;
+            }
+        }
+
+        if (params.all === true) {
+            // TODO: Must have atleast one other param
+            client.chatMessage(steamID, 'Updating pricelist...');
+
+            const pricelist = prices.getPricelist();
+
+            if (pricelist.length === 0) {
+                client.chatMessage(steamID, 'Your pricelist is empty');
+                return;
+            }
+
+            handlerManager.getHandler().cleanup();
+
+            for (i = 0; i < pricelist.length; i++) {
+                if (params.intent) {
```

Comment:
Check if intent is a number

Reply:
No i didn't add that because i moved the typeof for intent up